README.md

```diff
@@ -72,6 +72,24 @@ Divvy allows users to enter expenses as they are incurred, and calculates the be
 15. Click the "Show" button for "App Secret". Enter your Facebook password to display the value. Copy the shown value and into the `.env.js` file as the `FACEBOOK_SECRET`.
 16. With the dev server running, navigating to `http://localhost:3000/auth/facebook` redirects the browser to Facebook and ask for app authorization. Once authorized, the browser redirects to `http://localhost:3000/auth/facebook/callback` with a `code` query param (logged on the server). The browser then redirects to the application root.
 
+## Deployment (Heroku)
+
+1. Run `gulp build` in the command line to create a "dist" folder that will be deployed to Heroku.
+2. Navigate into the "dist" folder using the terminal.
+3. Create a deployment remote using `Heroku Create`.
+4. Use 'https://dashboard.heroku.com/apps' to view your new remote. You can change the app name at this step if necessary. You will have to update both the Heroku website name & the remote path name by using `git remote set-url heroku https://git.heroku/APPNAME.git`.
+4. Enter `heroku ps:scale web=1` into the command line to ensure at least one instance is running.
+5. Within the Heroku website, click on your project and navigate to "settings". Click on "Reveal config vars".
+6. Enter NODE_ENV, DOMAIN, SESSION_SECRET, FACEBOOK_ID, and FACEBOOK_SECRET.
+  a. Set NODE_ENV equal to "deployment".
+  b. Set DOMAIN equal to Heroku website link. Ensure there is no trailing "/".
```

Comment:
Set DOMAIN equal to the website link generated by Heroku. Ensure there is no trailing "/". Website name will be of format `https://git.heroku.com/APPNAME.git`